jenkins-pipelines/nemesis/longevity-5gb-1h-BlockNetworkMonkey-aws.jenkinsfile

```diff
@@ -7,6 +7,6 @@ longevityPipeline(
     backend: 'aws',
     region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: 'test-cases/nemesis/longevity-5gb-1h-BlockNetworkMonkey.yaml',
-    ip_ssh_connections: 'public'
```

Comment:
you are loosing this `ip_ssh_connections: 'public'`

and those won't work without it

Reply:
Fixed, added param expansion to the template (for now only supports string parameters)